apps/ide/src/plugins/project-wizard/build/build.js

```diff
@@ -14,11 +14,20 @@
  * limitations under the License.
  */
 
+/**
+ * @file Build runner
+ * @since 1.0.0
+ * @author cimfalab@gmail.com
```

Comment:
@wcho @hwshim For filling the `@author` value, I found the emails from the result of `git blame` against the old git repository. I'll change it.